### PR TITLE
Small changes for the release

### PR DESCRIPTION
**Pull request by abergeron**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -87,6 +87,7 @@ set_target_properties(gpuarray PROPERTIES
   COMPILE_FLAGS "-DGPUARRAY_BUILDING_DLL -DGPUARRAY_SHARED"
   INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib
   MACOSX_RPATH OFF
+  VERSION 0.0
```
Comment (nouiz) [inline src/CMakeLists.txt]: How we can get access to that version in Theano? Does this version if an interface change or any code version upgrade? I think we need to keep that information available in the version in some way.

Comment (abergeron) [inline src/CMakeLists.txt]: Normally this automatically handled through shared library versioning.

If you change an interface, you bump the first number, if you add new things you bump the second number.

**Comment by nouiz** [comment]
but the PR on theano just remove all versioning stuff. It should use the
new versioning interface instead.

**Comment by abergeron** [comment]
The shared library linker of the OS will handle this if we put the versioning in the library like this.

This is why we don't need custom handling in Theano anymore.

**Comment by nouiz** [comment]
Test for the set strides are missing.

**Comment by abergeron** [comment]
There is a test now (and the same checks that numpy does).

**Comment by nouiz** [comment]
Except for the docstring, we can merge this PR.

**Pull request merged**